Fix the comments w.r.t. configuration file usage with the -T option.
This change has been submitted upstream.

--- snort-2.9.9.0/snort.8.orig	2017-01-25 12:23:22.383189188 +0000
+++ snort-2.9.9.0/snort.8	2017-01-25 12:23:56.475130490 +0000
@@ -341,8 +341,7 @@
 indicating that everything is ready to proceed.  This is a good
 switch to use if daemon mode is going to be used, it verifies that
 the Snort configuration that is about to be used is valid and won't fail at
-run time. Note, Snort looks for either /etc/snort.conf or ./snort.conf.
-If your config lives elsewhere, use the -c option to specify a valid
+run time. Note that you will need to use the -c option to specify a valid
 .I config-file.
 .IP "-u user"
 Change the user/UID Snort runs under to
